compatibility_lib/compatibility_lib/test_package_crawler.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ast
+import os
+import unittest
+
+from compatibility_lib.package_crawler_static import get_package_info, get_module_info
+
+
+class TestSimplePackages(unittest.TestCase):
```

Comment:
We will need to make the unit tests actually testing each function, not testing the tool as a whole thing. And we should have more complex test data to ensure we cover as many edge cases as possible.

Reply:
I think that having gray box tests makes sense for this project. I would structure my test cases to handle concrete examples. Agreed that we need a lot of test examples to validate correct functionality.